wiremock-junit4/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id("wiremock.common-conventions")
+}
+
+dependencies {
+    api(project(":wiremock-common"))
+
+    api(libs.junit4) {
+        exclude(group = "org.hamcrest", module = "hamcrest-core")
```

Comment:
I wonder if we should remove this exclusion now you are consciously depending on JUnit 4 rather than accidentally bringing it in with WireMock?

Or replace it with this:
 and bring in this:
```
    modules {
        module("org.hamcrest:hamcrest-core") {
            replacedBy("org.hamcrest:hamcrest")
        }
        module("org.hamcrest:hamcrest-library") {
            replacedBy("org.hamcrest:hamcrest")
        }
    }
```
since they have now merged `library` and `core` into the one module?

Reply:
I guess it's up to the user to whether they want to use the depended-on hamcrest so we could just remove the exclusion and leave it at that.